yt/frontends/ramses/field_handlers.py

```diff
@@ -19,49 +20,31 @@ def register_field_handler(ph):
     FIELD_HANDLERS.add(ph)
 
 
-PRESENT_FIELD_FILES = {}
 DETECTED_FIELDS = {}
 
 
-class RAMSESFieldFileHandlerRegistry(type):
+class RegisteredRAMSESFieldFileHandler(abc.ABCMeta):
```

Comment:
Seems like there's a better approach now, may be worth joining efforts with @matthewturk !
see https://github.com/yt-project/yt/pull/2823